drydock/templates/drydock/k8s/ingress/static-cache.yml

```diff
@@ -0,0 +1,26 @@
+{%- for service, config in DRYDOCK_NGINX_STATIC_CACHE_CONFIG.items() %}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-static-{{ service }}
+  namespace: {{ K8S_NAMESPACE }}
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-body-size: 8m
+    nginx.ingress.kubernetes.io/proxy-buffering: "on"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      {{ patch("static-cache-config") | indent(6)}}
```

Comment:
What about adding the content of the file [drydock/patches/static-cache-config](https://github.com/eduNEXT/drydock/pull/106/files#diff-9ba3d1cc82bcdb839c9d60c984b1939023f52f9fb93c8cab0eca43a83f328eb6) here and add an `extra-static-cache-config` patch which is up to the plugin user to implement?